src/components/GenerateGifForm.js

```diff
@@ -21,15 +28,50 @@ class GenerateGifForm extends Component {
       case 'fontColor':
         this.props.updateTextColor(evt.target.value);
         break;
+
       case 'name':
         this.props.updateGIFFileName(evt.target.value);
         break;
+
+      case 'placement':
+        let values = evt.target.value.split('-');
+        let positionObj = {
+          textAlign: values[1],
+          textBaseline: values[0]
+        };
+        this.props.updateTextPosition(positionObj);
+        let selectValue = document.getElementById('GenerateGifForm-select');
```

Comment:
What is the reason for this -- ie using getElementId rather than accessing the value in the state that's associated with the input?

Reply:
No reason, just fixed